### PR TITLE
[Docker] Updating dependency on Mxnet-mkl

### PR DESCRIPTION
**Pull request by shoubhik**
We need dependency on mxnet mkl package to be able to write tests for end-to-end quantized networks.

### CODE REVIEW

**Comment by masahi** [comment]
the CI is not updated when you update docker. https://docs.tvm.ai/contribute/pull_request.html#ci-environment

so you might as well add pytorch and torhvision update here too.

**Comment by shoubhik** [comment]
> the CI is not updated when you update docker. https://docs.tvm.ai/contribute/pull_request.html#ci-environment
> 
When I look at prebuilt images [here](https://hub.docker.com/r/tvmai/ci-gpu/tags) which runs the front end tests, it is built on `linux/amd64`. Any particular reason it is on amd64? the image i build, should it be on amd64 too?

**Comment by tqchen** [comment]
I will merge this PR first, and will report back once we have updated the CI image

**Comment by tqchen** [comment]
The docker can be built on any env. @masahi can you send another PR for pytorch update? 

Note that we still need to confirm that all test-cases passes, as there might be problem during CI image update due to breaking dep on upstream. I will be traveling in the incoming week, but will try to keep things updated.

**Comment by masahi** [comment]
Sent a torch update to #4756. I confirmed locally that onnx/test_forward.py, the only place we use torch, works with the updated version.

**Comment by shoubhik** [comment]
I have also updated the tag in Jekins file and the gpu image file.

File: docker/Dockerfile.ci_gpu [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 # CI docker GPU env
-# tag: v0.56
+# tag: v0.57
```
Comment (tqchen) [inline docker/Dockerfile.ci_gpu]: we want to avoid updating the tag before build and test the correctness. I will send a separate PR to do so

Comment (shoubhik) [reply docker/Dockerfile.ci_gpu]: got it, i'll revert the changes.

**Comment by tqchen** [comment]
NOTE: when building the docker image, 

I get ERROR: mxnet_mkl-1.5.1-py2.py3-none-manylinux1_x86_64.whl is not a supported wheel on this platform.
This is an interesting error that might has things to do with wheel not recognized in the current docker setup. We will need to fix this and make sure the docker builds properly before proceed

**Comment by tqchen** [comment]
Update: seems was due to latest version of pip, will investigate more

**Comment by tqchen** [comment]
Update: blocked by https://github.com/apache/incubator-tvm/issues/4764

**Pull request closed without merging**